ironfish-cli/src/commands/start.ts

```diff
@@ -110,6 +110,11 @@ export default class Start extends IronfishCommand {
       description:
         'Path to a JSON file containing the network definition of a custom network to connect to',
     }),
+    enableWallet: Flags.boolean({
+      allowNo: true,
+      default: false,
```

Comment:
Should this be true by default? If it's `false`, then the current behavior won't hold true anymore.

Reply:
Done, sorry about that. I should have been more careful